aequilibrae/paths/traffic_assignment.py

```diff
@@ -29,7 +29,7 @@ class TrafficAssignment(object):
         demand.computational_view(['matrix']) # We will only assign one user class stored as 'matrix' inside the OMX file
 
         project = Project()
```

Comment:
I'd always be inclined to use construction to do your init/load ([RAII](https://en.wikipedia.org/wiki/Resource_acquisition_is_initialization))

so setting up to use either
```python
project = Project(project_dir)
```

or 

```python
project = Project.from_dir(project_dir)
```

Reply:
It used to be like that, but the API design around the creation of new projects and opening of existing ones turned out to be kinda weird. Any suggestions?